gov2/dynamodb/README.md

```diff
@@ -76,21 +61,6 @@ go mod tidy
 go run ./cmd -scenario movieTable
 ```
 
-### Query a table using PartiQL
-
-These two scenarios show you how to run PartiQL statements to query a DynamoDB table of
-movie data. You can do this one at a time or in batches. Both scenarios add, get,
-update, and delete movies in the table.
-
-Install all required resources and start the example by running one of the following in 
-the `dynamodb` folder at a command prompt.
-
-```
-go mod tidy
-go run ./cmd -scenario partiQLSingle
-go run ./cmd -scenario partiQLBatch
-```
-
```

Comment:
Keep these breakout descriptions of individual examples in the README.

Reply:
restored